### PR TITLE
Add coverity scan based on run-lcg-view action

### PR DESCRIPTION
**Pull request by petricm**
BEGINRELEASENOTES
- Add coverity nightly scan based on `run-lcg-view` action

ENDRELEASENOTES

I have been working on the action, and used podio as an example. @andresailer can give access to the project to whom it is needed.

### CODE REVIEW

**Comment by petricm** [comment]
As info for fine tuning the cron 

> What is the frequency for build submissions to Coverity Scan?
>
>The number of weekly builds per project are as follows:
>
>    Up to 28 builds per week, with a maximum of 4 builds per day, for projects with fewer than 100K lines of code
>    Up to 21 builds per week, with a maximum of 3 builds per day, for projects with 100K to 500K lines of code
>    Up to 14 builds per week, with a maximum of 2 build per day, for projects with 500K to 1 million lines of code
>    Up to 7 builds per week, with a maximum of 1 build per day, for projects with more than 1 million lines of code
>
>Once a project reaches the maximum builds per week, additional build requests will be rejected. You will be able to re-submit the build request the following week.
```

**Comment by petricm** [comment]
and if you want to see how nicely this works with [DD4hep](https://github.com/AIDASoft/DD4hep/runs/1952730965?check_suite_focus=true)

**Comment by andresailer** [comment]
The results are here: https://scan.coverity.com/projects/aidasoft-podio?tab=overview

File: .github/workflows/coverity.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+jobs:
+  run-coverity:
+    runs-on: ubuntu-latest
+    if: github.repository == 'AIDASoft/podio'
+    steps:
+    - uses: actions/checkout@v2
+    - uses: cvmfs-contrib/github-action-cvmfs@v2
+    - uses: aidasoft/run-lcg-view@v2
+      with:
+        coverity-cmake-command: 'cmake -DCMAKE_CXX_STANDARD=17 ..'
```
Comment (andresailer) [inline .github/workflows/coverity.yml:16]:
```suggestion
        coverity-cmake-command: 'cmake -DCMAKE_CXX_STANDARD=17  -DENABLE_SIO=ON ..'
```
This should work with LCG_99

**Pull request merged**